src/artwork.c

```diff
@@ -392,6 +393,64 @@ static struct online_source musicbrainz_source =
   };
 
 
+/* -------------------------------- HELPERS -------------------------------- */
+
+#define TMP_BUF_SIZE 1024
```

Comment:
Move up to other defines (if you have the define next to where it is used then there isn't much point in having it?) - or remove it entirely, I don't think you need it - see below

Reply:
Good point.  It's used 3 times but all in very close proximity so not really needed.